maven/org.eclipse.emf.mwe2.repository/category.xml

```diff
@@ -6,6 +6,8 @@
    <feature id="org.eclipse.emf.mwe.sdk" version="0.0.0">
       <category name="mwe2lang"/>
    </feature>
+   <bundle id="com.google.guava" version="27.1.0.qualifier"/>
```

Comment:
I doubt that ".qualifier" should be added here. Better use the qualified version, as in Xtext.

Reply:
this is a 1:1 copy from xtext

Reply:
OK, then it should be fine.